book/starting-utils.md

```diff
@@ -49,21 +49,33 @@ Usage:
 Specific information about what these commands do is given in the following sections and on the [*Solo CLI* README](https://github.com/3drobotics/solo-cli).
 
 
-## Connecting to Solo and the Internet
+## Connecting Solo to the Internet
 
-During development, you will most often want access to Solo and the Internet simultaneously. The command `solo wifi` connects your Controller to a local wifi network, allowing you to develop for Solo while also enabling connection to the outside Internet. The steps for doing so are as follows:
+The `solo wifi` command connects your Controller to a local WiFi network. Solo uses this connection (via the Controller network) to access the Internet during development and to [download and install packages](starting-installing.html#installing-packages).
```

Comment:
I would change "connects your Controller to a local WiFi network" to "connects your Controller to your WiFi network". I don't know if it's because English is not my native language, but connecting to "a local WiFi network" seems vague to me.